packages/mco/components/modals/app-manage-policies/policy-list-view.tsx

```diff
@@ -137,6 +142,30 @@ export const PolicyListView: React.FC<PolicyListViewProps> = ({
       payload: selectedPolicies,
     });
 
+  const [discoveredApps, discoveredAppsLoaded, discoveredAppsError] =
+    useK8sWatchResource<DRPlacementControlKind[]>(
+      getDRPlacementControlResourceObj({
+        // ToDo: Update this namespace with correct imperative apps namespace
+        namespace: ODFMCO_OPERATOR_NAMESPACE,
+      })
+    );
+
+  const isProtectedUnderNamespace = () => {
+    return discoveredApps.some((item) => {
+      const eligibleNamespaces = item.spec.eligibleForProtectionNamespaces.name
+        .split(',')
+        .map((val) => val.trim());
+      const primaryProtectedNamespaces =
+        item.spec.PrimaryProtectedNamespace.name
+          .split(',')
+          .map((val) => val.trim());
+      return (
+        eligibleNamespaces.includes(namespace) ||
+        primaryProtectedNamespaces.includes(namespace)
```

Comment:
```suggestion
      return (
        eligibleNamespaces.includes(namespace) 
```
Compare the policy ref with eligible policies.